Misc/NEWS.d/next/Core and Builtins/2021-07-18-19-32-38.bpo-44653.lu99mE.rst

```diff
@@ -0,0 +1 @@
+Support ``typing.Union`` in parameter substitution of the union type.
```

Comment:
Maybe this is more specific:
```suggestion
Flatten ``typing.Union`` during parameter substitution of the union type.
```

Reply:
Is it better? Before this PR `typing.Union` was not supported at all.

Reply:
Hmm oh right I forgot. I think it's good to also mention that the arguments are flattened though, since none of the other types are treated like that. Some people might expect ``(int | T)[Union[str, int]]`` to become ``int | Union[str, int]``.

```suggestion
Support ``typing.Union`` in parameter substitution of the union type.
Arguments in ``typing.Union`` are flattened and merged into the
union type.
```